CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 5.2.0
+- Upgrade from Symfony 6.4 to Symfony 7.3
+- Maintenance release, Update tooling, packages
```

Comment:
If possible, convention asks for a PR reference to each feature/maintenance task in each release.

Also, speculating, should this not be a major release as we are upgrading to a new major version of Symfony? Or maybe no significant platform requirements changed?  In that case this can remain a minor 👍 

For example; the user of this version needs to update some config; which can be considered a breaking change unless the config update is non breaking.